packages/http/src/mocker/HttpMocker.ts

```diff
@@ -33,7 +33,7 @@ export class HttpMocker implements IMocker<IHttpOperation, IHttpRequest, IHttpCo
     }
 
     // setting default values
-    const inputMediaType = input.data.headers && caseless(input.data.headers).get('content-type');
```

Comment:
The variable name was correct before, it was just being used poorly. The content-type is the inputMediaType. 

Maybe we could rename the variable to acceptHeader so there is no confusion. It might not be a single media type string there could be multiple values, so better to be clear about it being an Accept header.

Reply:
Done!